### DISCUSSION TITLE
Errors in the operation of the active calibration plugin with the sensor Sibionics

### DISCUSSION THREAD

**Discussion by Inkalika** [NightscoutFoundation/xDrip#3189] [Bug reports]
xDrip version 23.09.23
I'm using a Sibionics sensor.
It works with the modified app Sugar Joy+ that can be downloaded from here https://t.me/kubarev_modz
In xDrip set the data source to 640g/Eversense.
Since this sensor can only be calibrated when the active calibration plagin is turned on (the letter P in a circle is displayed next to the sugar value), after turning it on, the following errors were detected:

Raw readings are sent to Nightscout, not calibrated ones (after using the plugin)
If you delete calibrations on the phone Master or do calibrations. then instead of the sugar level, followers will display “???” until the initial calibration is completed. If 2 followers are enabled via the xdrip follower source, then this error can occur at any time and can be corrected by deleting one of the followers.
Is it possible to fix these functions when the active calibration plugin is enabled?

**Comment by Navid200** [comment]
For my benefit, please tell me about this sensor.  How does a user who uses the manufacturer's app calibrate it?

**Comment by Inkalika** [comment]
Calibration is not provided in the manufacturer's application. But the Libre sensor can be calibrated without using a plugin. But if this sensor is calibrated in this way, then calibrations are not applied. But if you enable the plugin, they are applied.

**Reply by Navid200** [reply]
So, the manufacturer claims that the sensors are perfect and need no calibration?

**Reply by Inkalika** [reply]
Yes, but in fact he makes a lot of mistakes, especially in the first few days. Then he usually lowers it by 1-1.5 mmol

**Reply by Navid200** [reply]
xDrip does not provide any means for you to calibrate a Dexcom G6 or G7 either other than just passing the calibration to the Dexcom device.  I suppose one can choose to use a plugin with Dexcom.  I am  not sure about that.  But, if someone did, I don't think we are going to make any changes to xDrip to accommodate that.

I can be corrected.  @jamorham

**Comment by aptash** [comment]
I think this issue is related: https://github.com/NightscoutFoundation/xDrip/discussions/3174
